vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/verdict.rb

```diff
@@ -0,0 +1,26 @@
+module SecureRooms
+
+  module AccessRules
+
+    class Verdict
```

Comment:
they feel so similar to me right now, I'd prefer to wait and see on this. it does seems like it might become much more useful if we add just a few more rules eventually